assembl/models/discussion.py

```diff
@@ -359,18 +359,28 @@ def get_discussion_urls(self):
 
     def check_authorized_email(self, user):
         # Check if the user has a verified email from a required domain
+        from .social_auth import SocialAuthAccount
         require_email_domain = self.preferences['require_email_domain']
-        if not require_email_domain:
+        autologin_backend = self.preferences['authorization_server_backend']
```

Comment:
Is there anywhere in the discussion preferences or in a documentation page where we can describe the shape and content of this pref? If it was an ini thing, there would be a clear trail of what this is supposed to entail, for example...